pkg/k8s/apis/cilium.io/v2/client/register.go

```diff
@@ -106,6 +106,26 @@ func CreateCustomResourceDefinitions(clientset apiextensionsclient.Interface) er
 	return g.Wait()
 }
 
+// CreateCustomResourceDefinitionFromBytes creates the given CRD in the
+// Kubernetes cluster. The given CRD is expected to be serialized into bytes,
+// from its YAML structure.
+func CreateCustomResourceDefinitionFromBytes(
```

Comment:
We may want to expose this via an interface that the `Daemon` implements. Something like this - https://github.com/cilium/cilium/blob/master/pkg/hubble/parser/getters/getters.go